### PR TITLE
Added : cart.html

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
Added cart page please review it.....
image

can visit live at : https://nostalgic-bell-c8dae0.netlify.app/cart.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            <div class="single">
+                <label for="search" style="color: white;  padding: .5rem; " class="fa fa-search fa-2x"></label>
+                <input type="search" name="search" id="search1" placeholder="Search">
+            </div>
+        
+        <div class="top-navbar-menu">
+            <a href="signin.html" class="top-menu home"><button class="btn">Login</button></a> &nbsp;&nbsp;&nbsp;
+            <a class="top-menu" href="cart.html"><i class="fa fa-shopping-cart fa-2x"><span>2</span></i></a> &nbsp;&nbsp;
+            <a  class="top-menu" href=""><i class="fa fa-heart fa-2x"><span>4</span></i></a>
+            &nbsp;&nbsp;&nbsp;
```
Comment (someshtiwarii) [inline index.html:27]: avoid using nbsp

File: style.css [hunk 2]
```diff
@@ -0,0 +1,174 @@
+.navigation-bar{
+    gap:60%;
+}
+
+body{
+    background: rgb(224, 220, 220);
```
Comment (someshtiwarii) [inline style.css:6]: use color variables please

File: cart.html [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+       <h4>
+       By :- ADIDAS  <span class="rate"> 4.8 &nbsp;<i class="fa fa-star"></i></span>
+       </h4>
+       <p>
+        Men's sport Running Shoes
+       <br><br>
+       <span class="price"> ₹1,499</span> <i> <strike>₹1,999 </strike>18% off </i>
+                         </p>
+                         <div class="card-btn">
+                             <div><button><li class="fa fa-minus"></li></button><button style="margin:8px; padding:.4rem 1rem;">1</button> <button><li class="fa fa-plus"></li></button></div>
```
Comment (someshtiwarii) [inline cart.html:53]: avoid inline styling

File: cart.html [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+</head>
+<body>
+    <!-- Navbar -->
+    <nav class="top-navbar">
+        <div class="top-navbar-icon">
+            <img src="https://bit.ly/3v2WZoY"/>
+            <div class="title">MetaKart</div>
+        </div>
+            <div class="single">
+                <label for="search" style="color: white;  padding: .5rem; " class="fa fa-search fa-2x"></label>
```
Comment (someshtiwarii) [inline cart.html:19]: you can give this label a class and then apply style to it (avoid inline styling)

**Review by someshtiwarii — APPROVED** [review]
just make the mentioned changes , rest looks fine !

File: cart.html (hunk 4 above)
Comment (Amandeep-sinha) [reply cart.html:19]: Yeah....

**Pull request merged**